Documentation/ceph-csi-drivers.md

```diff
@@ -6,18 +6,19 @@ indent: true
 {% include_relative branch.liquid %}
 # Ceph CSI Drivers
 
-There are two CSI drivers integrated with Rook that will enable different scenarios:
-
-* RBD: This driver is optimized for RWO pod access where only one pod may access the storage
-* CephFS: This driver allows for RWX with one or more pods accessing the same storage
-
-The drivers are enabled automatically with the Rook operator. They will be started
-in the same namespace as the operator when the first CephCluster CR is created.
-
-For documentation on consuming the storage:
-
-* RBD: See the [Block Storage](ceph-block.md) topic
-* CephFS: See the [Shared Filesystem](ceph-filesystem.md) topic
```

Comment:
There were two instances of the list "RBD, CephFS, NFS" that I combined. Does it still read well?

Reply:
definitely looks better combined, thanks